controller/slot_filling.py

```diff
@@ -159,22 +171,94 @@ def mapAChead(acHead):
         return 15
```

Comment:
return from the constant instead of hardcoded number.

Reply:
Done

Reply:
Done